Client/Frontend/Browser/Toolbars/BottomToolbar/Menu/MenuViewController.swift

```diff
@@ -1,405 +1,287 @@
+// Copyright 2020 The Brave Authors. All rights reserved.
 // This Source Code Form is subject to the terms of the Mozilla Public
 // License, v. 2.0. If a copy of the MPL was not distributed with this
 // file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
 import UIKit
-import Storage
+import PanModal
 import Shared
 import BraveShared
 import BraveUI
+import SwiftUI
 
-private class MenuCell: UITableViewCell {
-    let iconView = UIImageView().then {
-        $0.snp.makeConstraints { make in
-            make.width.equalTo(50)
+struct MenuItemHeaderView: View {
+    @Environment(\.colorScheme) var colorScheme: ColorScheme
```

Comment:
This does not seem used here? and maybe make private if it is needed to be here

Reply:
These are used to force SwiftUI to recompute the body of the View when the users system theme changes (as we don't have an observable "current" theme object anywhere)